dts/bindings/sensor/ti,tmp112.yaml

```diff
@@ -19,6 +19,17 @@ properties:
       - 1000
       - 4000
       - 8000
+
   extended-mode:
-    description: When true use 13-bit data format allowing measuring temperature up to 128°C
+    description: When true use 13-bit data format allowing measuring temperature up to 150°C
     type: boolean
+
+  t-low-micro-c:
+    description: Sets the default tLow threshold in micro-Celsius.
+    type: int
+    default: 75000000
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules

Reply:
Done, thanks!